### ISSUE TITLE
Lack of Internet Explorer support is not mentioned on the documentation page

### ISSUE THREAD

**Issue by tylerlweb** [limonte/sweetalert2#536]
Many people still need to support IE and you wouldn't know that IE isn't supported by looking at the documentation page. This should be plainly stated.

**Comment by taxpayer** [comment]
Oh, yes, it does...

https://github.com/limonte/sweetalert2#browser-compatibility

**Comment by tylerlweb** [comment]
Sorry, I should've been more clear - I'm talking about this page, which is the page that a lot of us focus on when checking it out:

https://limonte.github.io/sweetalert2/

**Comment by taxpayer** [comment]
Ok, I got your point. But I usually focus on the Github pages on the projects I use, that's why I promptly sent you the link above. I also recommend you to do the same although, as I told before, I got your point.

**Comment by limonte** [comment]
1. IE11 **is** supported
2. https://limonte.github.io/sweetalert2/ is the marketing page, https://github.com/limonte/sweetalert2#browser-compatibility is more detailed and technical page.

**Comment by tylerlweb** [comment]
I think this is more of a legitimate issue than you realize. People don't come to the SweetAlert pages with a preconception of "this is the marketing page" and "this is the more detailed and technical page," especially when they both list tons of usage details. In my case, I didn't realize (coming from SweetAlert 1) that SweetAlert 2 dropped support for IE 8/9/10, so I rewrote the swal()'s in my app - a couple weeks later, I realized IE wasn't supported and I had to revert all that code back to what it was. For those of us who are STILL forced to support these horrible browsers, it's a good idea to make it clear on the "marketing" page too. It doesn't have to be that prominent - as long as it's visible on the page somewhere, it'll prevent other people from falling into the same trap.
